ciao-4.9/contrib/param/blanksky.par

```diff
@@ -4,6 +4,7 @@ asolfile,f,h,"",,,"Source aspect solution file(s)"
 weight_method,s,h,"particle",particle|time,,"Method to calculate background scale factor"
 bkgparams,s,h,"default",,,"Optional argument for background scaling, default is [pi=300:500] for HRC and [energy=9000:12000] for ACIS"
 tmpdir,s,h,"${ASCDS_WORK_PATH}",,,"Directory for temporary files"
-clobber,b,h,no,,,"OK to overwrite existing output file?"
+random,i,h,0,0,,"Random seed, 0=clock time"
+clobber,b,h,yes,,,"OK to overwrite existing output file?"
```

Comment:
no, I'll revert it back to clobber=no, and update this along with a typo fix I noticed in the main script